src/content/docs/devices.md

```
@@ -1,41 +1,38 @@
 
 | Name                 | Links                                       | Notes                                                                |
 |:---------------------|:--------------------------------------------|:---------------------------------------------------------------------|
-| Android Phones       |                                             | [Must install an app](/devices/android)
-| iPhones              |                                             | [Pair to retrieve IRK](/devices/apple)
-| Apple Watches        |                                             | [Use iCloud keychain to get IRK](/devices/apple)
-| Wear OS Smartwatches |                                             | [Must use HA companion app](/devices/android)
-| Tiles                | [amz/us](https://amzn.to/3h77T5f){:target="_blank"}           | These work great, but update somewhat slow
-| Blue Charm Beacons   | [amz/us](https://amzn.to/2YGdA3w){:target="_blank"}           | These work great [^beaconconfig]
-| Blue Charm Beacons (BC021-MultiBeacon) | [amz/us](https://amzn.to/4ceo8Jb){:target="_blank"}           | Bluetooth BLE 5.0 with Movement Sensor [^beaconconfig]
-| Blue Charm Beacons (BC05-MultiBeacon) | [amz/us](https://amzn.to/4iWu0Ja){:target="_blank"}           | Long Range Water-Resistant BLE iBeacon with Adjustable Movement - Ideal for mail detection [^beaconconfig]
+| Android Phones       |                                             | [Must install an app](/android)
```

markdownlint-cli2 (0.20.0)

[warning] Table pipe style
Expected: leading_and_trailing; Actual: leading_only; Missing trailing pipe

(MD055, table-pipe-style)